driver-service/src/test/java/com/dulfinne/taxi/driverservice/util/CarTestData.java

```diff
@@ -0,0 +1,70 @@
+package com.dulfinne.taxi.driverservice.util;
+
+import com.dulfinne.taxi.driverservice.dto.request.CarRequest;
+import com.dulfinne.taxi.driverservice.dto.response.CarResponse;
+import com.dulfinne.taxi.driverservice.model.Car;
+import com.dulfinne.taxi.driverservice.model.CarCategory;
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class CarTestData {
+
+  public static final Long ID = 1L;
+  public static final String NAME = "BMW";
+  public static final String REGISTRATION_NUMBER = "1234AA-5";
+  public static final CarCategory CAR_CATEGORY = CarCategory.ECONOM;
+  public static final String COLOR = "Black";
+
+  public static final String UPDATED_NAME = "LADA";
+  public static final String UPDATED_REGISTRATION_NUMBER = "1234AA-6";
+  public static final CarCategory UPDATED_CAR_CATEGORY = CarCategory.COMFORT;
+  public static final String UPDATED_COLOR = "Velvet";
+
+  public Car getCar() {
+    return Car.builder()
+            .id(ID)
+            .name(NAME)
+            .registrationNumber(REGISTRATION_NUMBER)
+            .carCategory(CAR_CATEGORY)
+            .color(COLOR)
+            .build();
+  }
```

Comment:
Можно возвращать билдер с дефолтными значениями полей и вызывать у него метод build() уже там, где настроенный объект будет использоваться. Это позволяет взять объект с дефолтными значениями полей и поменять только некоторые поля, если есть такая необходимость. Названия методов менять не надо.
```suggestion
  public CarBuilder getCar() {
    return Car.builder()
            .id(ID)
            .name(NAME)
            .registrationNumber(REGISTRATION_NUMBER)
            .carCategory(CAR_CATEGORY)
            .color(COLOR);
  }
```